Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "vm-virtio"
+version = "0.1.0"
+authors = ["The Chromium OS Authors"]
+description = "virtio specification implementation"
+repository = "https://github.com/rust-vmm/vm-virtio"
+keywords = ["virtio"]
+readme = "README.md"
+license = "Apache-2.0 OR MIT"
+
+[dependencies]
+byteorder = "=1.2.1"
+epoll = "=4.0.1"
+libc = ">=0.2.39"
+log = "*"
+vmm-sys-util = { git = "https://github.com/liujing2/vmm-sys-util" }
+tempfile = ">=3.0.2"
```

Comment:
I don't think you need this dependency. Also in the future I would not use tempfile because it pulls a lot of weird dependencies like cloudabi, winapi-x86_64-pc-windows-gnu and such.

Reply:
Thanks for spotting that one. Fixed now.